src/pykiso/lib/auxiliaries/communication_auxiliary.py

```diff
@@ -114,19 +143,32 @@ def run_command(
         return state
 
     def receive_message(
-        self, blocking: bool = True, timeout_in_s: float = None
+        self,
+        blocking: bool = True,
+        timeout_in_s: float = None,
+        clear_buffer: bool = False,
```

Comment:
Do we still need this API? Now that we do not loop on the receive, the queue should always be empty now

Reply:
Is there any constellation where the queue could not be empty?